From 5022d0a23bce9e5e475f1fcc6dabba29dd7e6d6e Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 18 Mar 2019 17:14:45 +0000
Subject: [PATCH] TOOLS-2101 jsstyle submodules too old for old perls Reviewed
 by: Trent Mick <trentm@gmail.com> Reviewed by: Cody Peter Mello
 <melloc@writev.io> Approved by: Cody Peter Mello <melloc@writev.io>

---
 .gitmodules  | 2 +-
 deps/jsstyle | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index c74f8e2..855677b 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -3,7 +3,7 @@
 	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle.git
+	url = https://github.com/joyent/jsstyle.git
 [submodule "deps/eng"]
 	path = deps/eng
 	url = https://github.com/joyent/eng.git
diff --git a/deps/jsstyle b/deps/jsstyle
index b94eb62..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit b94eb629ee524d21b563c8b4461caa2bdf1becc6
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
-- 
2.21.0

